website/project/model.py

```diff
@@ -615,6 +660,10 @@ class Node(GuidStoredObject, AddonModelMixin):
     # {<User.id>: [<Node._id>, <Node2._id>, ...] }
     child_node_subscriptions = fields.DictionaryField(default=dict)
 
+    # Files that contains comments and the number of comments each contains
+    # {<File1.id>: int, <File2.id>: int}
+    commented_files = fields.DictionaryField(default=dict)
```

Comment:
@samanehsan I forgot to tell you about this! This is a list of files that is commented on... it should be initialized as an empty dict for all current projects in the database